apps/babanews/src/components/website-builder-overwrites/blocks/teaser.tsx

```diff
@@ -0,0 +1,11 @@
+import {BuilderTeaserProps, Teaser} from '@wepublish/website'
+
+import {ListTeaser} from './list-teaser'
+
+export const BabanewsTeaser = (props: BuilderTeaserProps) => {
+  if (!props.numColumns) {
```

Comment:
Is this really what we want? Shouldn't this be solved in the block renderer?

Reply:
that's correct. I've removed it and moved the logic to block-renderer